### DISCUSSION TITLE
Fails to load image on wasm but not native ("Format error decoding Png: Invalid PNG signature., this is an error in `bevy_render`.")

### DISCUSSION THREAD

**Discussion by JustAnotherCodemonkey** [bevyengine/bevy#8853] [Q&A] [answered]
I'm getting into bevy as a method of creating web games and am having an issue loading a sprite onto the screen. When I run natively, it runs exactly as expected but when it comes to the browser, the sprite does not show up and in the console, I get the "warning" in the title.
Here's what the project looks like:
<img width="1904" alt="Screenshot 2023-06-15 at 2 18 27 PM" src="https://github.com/bevyengine/bevy/assets/131801775/a9b7d0f1-7fb7-4bd1-9b9f-fd09237e5751">
I use `trunk serve` as my server to run.

**Comment by mockersf** [comment]
Could you check in your browser console that it's correctly sending the file?

**Reply by vinnyhoward** [reply]
I am running into this exact same issue and haven't been able to solve it since yesterday. I can give you a screenshot of my browser console if that helps. Should be the same warning

![Screenshot 2023-06-15 at 3 58 15 PM](https://github.com/bevyengine/bevy/assets/31226035/7ba35339-e5c6-4710-a0f2-94d90c15607e)

**Reply by mockersf** [reply, marked as answer]
From your screenshot, your asset folder is not served by your http server. If you're using a normal http server, you should just copy the directory. if you're using trunk, you should take a look at https://trunkrs.dev/assets/

**Reply by vinnyhoward** [reply]
Ah okay thanks for the reply and quick response!
